biz.aQute.resolve/src/biz/aQute/resolve/RunResolution.java

```diff
@@ -208,7 +210,16 @@ public boolean updateBundles(BndEditModel model) {
 			newer = older;
 		}
 
-		model.setRunBundles(newer);
+		// Apply the -runbundles decorator on the computed RunBundles
```

Comment:
You have to change the check if the runbundles are identical before on line 197. The decoration might change the runbundles.

Reply:
I have moved the application of decoration above the older/newer check.